<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TruckingDialog</class>
 <widget class="QDialog" name="TruckingDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>272</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Trucking</string>
  </property>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Trucker</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3ListBox" name="truckerListBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>570</width>
     <height>127</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="savetruckers">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="truckingCancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="truckingDone">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="truckingDel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qprice">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Per</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Extended</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qqty">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qextend">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qtrucker">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="qtyLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Qty</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="Qunit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Load</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CWT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Head</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mile</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Qtrucker</tabstop>
  <tabstop>Qunit</tabstop>
  <tabstop>Qprice</tabstop>
  <tabstop>Qqty</tabstop>
  <tabstop>truckerListBox</tabstop>
  <tabstop>savetruckers</tabstop>
  <tabstop>truckingCancel</tabstop>
  <tabstop>truckingDel</tabstop>
  <tabstop>Qextend</tabstop>
  <tabstop>truckingDone</tabstop>
 </tabstops>
 <includes>
  <include location="local">dao/truckingrecorddao.h</include>
  <include location="global">vector</include>
  <include location="local">types/invoice.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
